Traceback (most recent call last):
  File "/home/runner/work/aioshumway/aioshumway/.tox/py/bin/py.test", line 5, in <module>
    from pytest import main
  File "/home/runner/work/aioshumway/aioshumway/.tox/py/lib/python3.6/site-packages/pytest.py", line 13, in <module>
    from _pytest.fixtures import fixture, yield_fixture
  File "/home/runner/work/aioshumway/aioshumway/.tox/py/lib/python3.6/site-packages/_pytest/fixtures.py", line 842, in <module>
    class FixtureFunctionMarker(object):
  File "/home/runner/work/aioshumway/aioshumway/.tox/py/lib/python3.6/site-packages/_pytest/fixtures.py", line 844, in FixtureFunctionMarker
    params = attr.ib(convert=attr.converters.optional(tuple))
TypeError: attrib() got an unexpected keyword argument 'convert'


How is this passing locally???

--- a/aioshumway/__init__.py
+++ b/aioshumway/__init__.py
@@ -20,7 +20,7 @@ with the async/await syntax.
 """
 
 __author__ = 'Matt Obarzanek'
-__version__ = '1.0.0'
+__version__ = '1.0.1'
 __license__ = 'Apache 2.0'
 __email__ = 'matto@spotify.com'
 __description__ = 'Asynchronous metrics library for ffwd; DEPRECATED'

